2/scss/big-slaider.scss

```diff
@@ -0,0 +1,107 @@
+@import "vars";
+@import "fonts";
+@import "mixins";
+
+.wrapper-big-slaider {
+  margin-top: 30px;
+  padding-top: 30px;
+  background: linear-gradient(15deg, #e0e0e0, #ffffff);
+  .big-slider-container {
+    min-height: 1085px;
+    max-width: 1440px;
+    margin: auto;
+    .slider-title-container {
+      text-align: center;
+      .slider-title{
+        @include text-size-standart;
+      }
+    }
+  }
+}
+
+.inner-wrapper {
+  .container-dots {
+    margin: 30px 0 30px 0;
+    .slick-dots {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      li {
+        margin: 0 10px 0 10px;
+        list-style: none;
+      }
+      button {
```

Comment:
this button has size 10*10px - it will be almost impossible to tap it with a finger :)

You could make buttons this size:
![image](https://user-images.githubusercontent.com/17281920/121788430-ebd4da80-cbd5-11eb-86ac-eaa308df1bbf.png)

Reply:
added paddings